tools/extras/install_kaldi.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# 
+# 
+# Installation script for Kaldi
+# 
+set -x -e
+
+if [[ "$1" =~ ^-?-h ]]; then
+  echo "Usage: $0 [KALDI_DIR]"
+  exit 1
+fi
+
+KALDI_DIR="$1"
```

Comment:
this script should install kaldi to current directory if no argument is provided.